### ISSUE TITLE
Game crash while starting

### ISSUE THREAD

**Issue by vanderlurch** [anegostudios/VintageStory-Issues#438]
**Game Version:** 1.12.5
**Platform:** Windows 10
**Modded:** No

### Description
Explain the issue you're running into.

### How to reproduce
1. Make a '....'
2. Use it on '....'
3. It crashes / doesn't do the thing.

### Expected behavior
If not obvious, what did you expect to happen?

### Screenshots
Attach a screenshot if it helps explain the issue / shows a visual bug.

### Logs
```
Paste crash here.

Running on 64 bit Windows with 16 GB RAM 
Version: v1.12.5 (Stable)05.03.2020 23:46:04: Critical error occurred
System.NullReferenceException: Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.
   bei Vintagestory.Client.NoObf.ClientPlatformWindows.LoadTexture(IBitmap bmp, Boolean linearMag, Int32 clampMode, Boolean generateMipmaps)
   bei _mKeEWUbBZsjLS6VlGWgOoYQFk2K._KNs6g2GMtsTu5gnL4MXdRgZVlLh()
   bei _lt8TgSVuBImgzFgAFlSeVzSFsSy._HdDsdRJDH8nMF88r278C4eqdINGA()
   bei _lt8TgSVuBImgzFgAFlSeVzSFsSy._gjiBClIE7Yspuz65gCqPGSAnUYeb()
   bei _lt8TgSVuBImgzFgAFlSeVzSFsSy._NbPATb9BXlxRc4ZhH2GgyFCUInp(Single )
   bei Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   bei System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   bei OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   bei OpenTK.GameWindow.DispatchRenderFrame()
   bei OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   bei _hCuHXesCXlp0LGSdrsPFpw4up5n._mOm8EuKIvHpaJoztxNWRpAoOpbM(_55ISULxdcpPfF8D2JP831aWhOBe , String[] )
   bei _iCfIbY24GyBXP5lAwqsSlD9COxC._mOm8EuKIvHpaJoztxNWRpAoOpbM(ThreadStart )
-------------------------------

```

**Comment by timklimowicz** [comment]
I've been getting the same thing the last few days when I've tried to play. I tried to reinstall the same version, but still getting the error.

Worth noting that I *did* successfully play this version for at least a handful of sessions, without issue. Most were single player in an ongoing world, and during another session I joined a public multiplayer server, and later that same session opened up my single-player world to let a friend join. I played the same world single player at least once after that.

```Running on 64 bit Windows with 24 GB RAM 
Version: v1.12.5 (Stable)3/5/2020 5:51:55 PM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.Client.NoObf.ClientPlatformWindows.LoadTexture(IBitmap bmp, Boolean linearMag, Int32 clampMode, Boolean generateMipmaps)
   at _mKeEWUbBZsjLS6VlGWgOoYQFk2K._KNs6g2GMtsTu5gnL4MXdRgZVlLh()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._HdDsdRJDH8nMF88r278C4eqdINGA()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._gjiBClIE7Yspuz65gCqPGSAnUYeb()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._NbPATb9BXlxRc4ZhH2GgyFCUInp(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _hCuHXesCXlp0LGSdrsPFpw4up5n._mOm8EuKIvHpaJoztxNWRpAoOpbM(_55ISULxdcpPfF8D2JP831aWhOBe , String[] )
   at _iCfIbY24GyBXP5lAwqsSlD9COxC._mOm8EuKIvHpaJoztxNWRpAoOpbM(ThreadStart )
-------------------------------
```

**Comment by soconnor73** [comment]
Same just started happening to me.  Game worked yesterday, now crashes on startup.  No mods.
Game Version: 1.12.5
Platform: Windows 10
Modded: No

```
Version: v1.12.5 (Stable)3/5/2020 6:09:02 PM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.Client.NoObf.ClientPlatformWindows.LoadTexture(IBitmap bmp, Boolean linearMag, Int32 clampMode, Boolean generateMipmaps)
   at _mKeEWUbBZsjLS6VlGWgOoYQFk2K._KNs6g2GMtsTu5gnL4MXdRgZVlLh()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._HdDsdRJDH8nMF88r278C4eqdINGA()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._gjiBClIE7Yspuz65gCqPGSAnUYeb()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._NbPATb9BXlxRc4ZhH2GgyFCUInp(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _hCuHXesCXlp0LGSdrsPFpw4up5n._mOm8EuKIvHpaJoztxNWRpAoOpbM(_55ISULxdcpPfF8D2JP831aWhOBe , String[] )
   at _iCfIbY24GyBXP5lAwqsSlD9COxC._mOm8EuKIvHpaJoztxNWRpAoOpbM(ThreadStart )
-------------------------------
```

**Comment by Rika58** [comment]
I've been getting the same thing the last few days when I've tried to play. I tried to reinstall the same version, but still getting the error.



Running on 64 bit Windows with 32 GB RAM 
Version: v1.12.5 (Stable)06.03.2020 02:16:07: Critical error occurred
System.NullReferenceException: Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.
   bei Vintagestory.Client.NoObf.ClientPlatformWindows.LoadTexture(IBitmap bmp, Boolean linearMag, Int32 clampMode, Boolean generateMipmaps)
   bei _mKeEWUbBZsjLS6VlGWgOoYQFk2K._KNs6g2GMtsTu5gnL4MXdRgZVlLh()
   bei _lt8TgSVuBImgzFgAFlSeVzSFsSy._HdDsdRJDH8nMF88r278C4eqdINGA()
   bei _lt8TgSVuBImgzFgAFlSeVzSFsSy._gjiBClIE7Yspuz65gCqPGSAnUYeb()
   bei _lt8TgSVuBImgzFgAFlSeVzSFsSy._NbPATb9BXlxRc4ZhH2GgyFCUInp(Single )
   bei Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   bei System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   bei OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   bei OpenTK.GameWindow.DispatchRenderFrame()
   bei OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   bei _hCuHXesCXlp0LGSdrsPFpw4up5n._mOm8EuKIvHpaJoztxNWRpAoOpbM(_55ISULxdcpPfF8D2JP831aWhOBe , String[] )
   bei _iCfIbY24GyBXP5lAwqsSlD9COxC._mOm8EuKIvHpaJoztxNWRpAoOpbM(ThreadStart )
-------------------------------

**Comment by cdkraft** [comment]
Worked fine yesterday. No change to setup. No mods. Same as others.  Crash on open.

Running on 64 bit Windows with 16 GB RAM 
Version: v1.12.5 (Stable)2020-03-05 5:38:04 PM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.Client.NoObf.ClientPlatformWindows.LoadTexture(IBitmap bmp, Boolean linearMag, Int32 clampMode, Boolean generateMipmaps)
   at _mKeEWUbBZsjLS6VlGWgOoYQFk2K._KNs6g2GMtsTu5gnL4MXdRgZVlLh()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._HdDsdRJDH8nMF88r278C4eqdINGA()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._gjiBClIE7Yspuz65gCqPGSAnUYeb()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._NbPATb9BXlxRc4ZhH2GgyFCUInp(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _hCuHXesCXlp0LGSdrsPFpw4up5n._mOm8EuKIvHpaJoztxNWRpAoOpbM(_55ISULxdcpPfF8D2JP831aWhOBe , String[] )
   at _iCfIbY24GyBXP5lAwqsSlD9COxC._mOm8EuKIvHpaJoztxNWRpAoOpbM(ThreadStart )
-------------------------------

**Comment by fierykilljoy** [comment]
i have this same problem it just crashes on me it played fine before even on multiplayer and hosting hope it gets fixed soon miss playing it.                                           Running on 64 bit Windows with 16 GB RAM 
Version: v1.12.5 (Stable)3/5/2020 8:18:55 PM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.Client.NoObf.ClientPlatformWindows.LoadTexture(IBitmap bmp, Boolean linearMag, Int32 clampMode, Boolean generateMipmaps)
   at _mKeEWUbBZsjLS6VlGWgOoYQFk2K._KNs6g2GMtsTu5gnL4MXdRgZVlLh()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._HdDsdRJDH8nMF88r278C4eqdINGA()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._gjiBClIE7Yspuz65gCqPGSAnUYeb()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._NbPATb9BXlxRc4ZhH2GgyFCUInp(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _hCuHXesCXlp0LGSdrsPFpw4up5n._mOm8EuKIvHpaJoztxNWRpAoOpbM(_55ISULxdcpPfF8D2JP831aWhOBe , String[] )
   at _iCfIbY24GyBXP5lAwqsSlD9COxC._mOm8EuKIvHpaJoztxNWRpAoOpbM(ThreadStart )
-------------------------------

**Comment by AAE-Animadversor** [comment]
Encountering the same error on startup as of tonight. Running single player only with no mods, worked without issue using the exact some configuration last night. Reinstalled to no effect.

Running on 64 bit Windows with 24 GB RAM 
Version: v1.12.5 (Stable)3/5/2020 9:33:25 PM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.Client.NoObf.ClientPlatformWindows.LoadTexture(IBitmap bmp, Boolean linearMag, Int32 clampMode, Boolean generateMipmaps)
   at _mKeEWUbBZsjLS6VlGWgOoYQFk2K._KNs6g2GMtsTu5gnL4MXdRgZVlLh()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._HdDsdRJDH8nMF88r278C4eqdINGA()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._gjiBClIE7Yspuz65gCqPGSAnUYeb()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._NbPATb9BXlxRc4ZhH2GgyFCUInp(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _hCuHXesCXlp0LGSdrsPFpw4up5n._mOm8EuKIvHpaJoztxNWRpAoOpbM(_55ISULxdcpPfF8D2JP831aWhOBe , String[] )
   at _iCfIbY24GyBXP5lAwqsSlD9COxC._mOm8EuKIvHpaJoztxNWRpAoOpbM(ThreadStart )

**Comment by copygirl** [comment]
Already reported in #407 and fixed for 1.12.6. Please update your game!
